Signal/src/call/WebRTCCallMessageHandler.swift

```diff
@@ -30,7 +31,7 @@ class WebRTCCallMessageHandler: NSObject, OWSCallMessageHandler {
         Logger.verbose("\(TAG) handling offer from caller:\(callerId)")
 
         let thread = TSContactThread.getOrCreateThread(contactId: callerId)
-        CallService.signalingQueue.async {
+        DispatchQueue.main.async {
```

Comment:
I would have guessed that this is already main thread. Can we get rid of the dispatch and just assertMainQueue?